nix/sources.json

```diff
@@ -12,39 +12,39 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "nixpkgs": {
-        "branch": "fix-yarn2nix-for-nix-2.5-6_29769d2a1390d294469bcc6518f17931953545e1",
+        "branch": "master",
         "description": "Nix Packages collection",
         "homepage": "",
-        "owner": "rhinofi",
+        "owner": "nixos",
         "repo": "nixpkgs",
-        "rev": "41b5736e8a7584ddf096a05b8b3afaf2a75e66fe",
-        "sha256": "0jg0jvc1acw9f71v0c7yyp13c37wjz2slnb3jd1d066g5vz3b3qi",
+        "rev": "70414d340baab99231000e4af02ec812e7d0c17b",
+        "sha256": "1zirk21zl3a7ycfafra8z6rrixlqxxyiygmrai49gl1xh9ffir71",
         "type": "tarball",
-        "url": "https://github.com/rhinofi/nixpkgs/archive/41b5736e8a7584ddf096a05b8b3afaf2a75e66fe.tar.gz",
+        "url": "https://github.com/nixos/nixpkgs/archive/70414d340baab99231000e4af02ec812e7d0c17b.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
-    "yarn-berry-cjs": {
-        "branch": "master",
-        "description": null,
+    "rhino-core": {
+        "branch": "dev",
+        "description": "Instructions to deploy the required services for DeversiFi Exchange",
         "homepage": "",
         "owner": "rhinofi",
-        "repo": "yarn-berry-cjs",
-        "rev": "75a5282d9606586d7c2f9e38b07f352cdeab46e3",
-        "sha256": "1hwxlglh02b2h6778dd24hfdhs765s6fg55lhy6ll730n75wxvyp",
+        "repo": "rhino-core",
+        "rev": "104a3209ea21bb1ee3d4a0de0a201a34635b6629",
+        "sha256": "1hff0p8d428cipcbfr444a4kk9cz9rx4lkmvlxaxhz2lm0mpwrn1",
         "type": "tarball",
-        "url": "https://github.com/rhinofi/yarn-berry-cjs/archive/75a5282d9606586d7c2f9e38b07f352cdeab46e3.tar.gz",
+        "url": "https://github.com/rhinofi/rhino-core/archive/104a3209ea21bb1ee3d4a0de0a201a34635b6629.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
-    "rhino-core": {
-        "branch": "dev",
-        "description": "Instructions to deploy the required services for DeversiFi Exchange",
+    "yarn-berry-cjs-rhinofi": {
```

Comment:
is this not exposed through rhino-core?

Reply:
default `shell.nix` doesn't depend on rhino-core since it's a private repo

I thought I would be nice if any external devs could us the default shell, so only included open deps in it.